core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -30,6 +30,7 @@ import org.apache.spark.annotation.DeveloperApi
 @DeveloperApi
 class TaskInfo(
     val taskId: Long,
+    /** the index of this task in its TaskSet.  *Not* necessarily the same as the partitionid */
```

Comment:
Ah ok got it.  I saw one of your earlier comments but wasn't sure how to interpret what was striked-through in it.  Since this is in the developer (so semi-user-facing), what about changing this comment to: "The index of this task within its task set. Not necessarily the same as the ID of the RDD partition that the task is computing."